unittests/test_notifications.py

```diff
@@ -53,4 +53,4 @@ def test_merge_notifications_list(self):
         self.assertEqual('alert' in merged_notifications.other, True)
         self.assertEqual('mail' in merged_notifications.other, True)
         self.assertEqual('slack' in merged_notifications.other, True)  # default alert from global
-        self.assertEqual(len(merged_notifications.other), 3)
+        self.assertEqual(len(merged_notifications.other), 4)
```

Comment:
I would think functionality would not change if doing a simple todo. Why does this value increase?

Reply:
This is also what I wrote in the comment above: _I have no clue why this unittest fails._

Reply:
merged_notifications.other is now ['slack', 'mail', 'alert', 'alert']
before that it was: ['slack', 'mail', 'alert']

Reply:
It is because of "default alert of global", see line 55 @Maffooch